doc/examples/gfal_copy.c

```diff
@@ -47,21 +47,24 @@ static void my_monitor_callback(gfalt_transfer_status_t h, const char* src,
         size_t inst = gfalt_copy_get_instant_baudrate(h, NULL);
         size_t trans = gfalt_copy_get_bytes_transfered(h, NULL);
         time_t elapsed = gfalt_copy_get_elapsed_time(h, NULL);
-
-        printf("%d bytes/second average (%d instant). Transferred %d, elapsed %d seconds", avg, inst, trans, elapsed);
+        printf("%d bytes/second average (%d instant). Transferred %d, elapsed %d seconds \n", avg, inst, trans, elapsed);
     }
 }
 
 
 int main(int argc, char** argv)
 {
+    printf("Main");
```

Comment:
I'd drop any unneeded code, like this...

Reply:
I'm of the same opinion. This example file should be left as it was originally, except for the addition of the `\n` symbol in the print statement.